bedrock/settings/base.py

```diff
@@ -584,7 +584,7 @@ def lazy_email_backend():
 }
 
 # Google Analytics
-GA_ACCOUNT_CODE = ''
+GA_ACCOUNT_CODE = 'UA-36116321-20'
```

Comment:
I think you copied this from my test, I left this in by accident when I was testing. I think this should be completely removed.

Double check this should be nix'd with paul tomorrow when things are less crazy.